src/System.Private.CoreLib/ILLinkTrim.xml

```diff
@@ -23,6 +23,9 @@
     <type fullname="System.Runtime.CompilerServices.AsyncMethodBuilderCore">
       <method name="TryGetStateMachineForDebugger" />
     </type>
+    <type fullname="System.Runtime.CompilerServices.AsyncIteratorMethodBuilder">
+      <property name="ObjectIdForDebugger" />
```

Comment:
would it be possible/worth it to combine all these into 
```
<type fullname=*">
  <property name="ObjectIdForDebugger" />
</type>
```

Reply:
or maybe `fullname=System.Runtime.CompilerServices.*`

Reply:
I'd be ok with the latter, but let's go with this for now, since it's just following the pattern already established and you've tested it.